src/remote/index.js

```diff
@@ -106,20 +101,10 @@ export default class Remote implements Side {
       _rev: created._rev
     }
 
+    // TODO do we use the returned values somewhere?
```

Comment:
I'm not sure either, but I would definitely prefer doing so instead of changing the given metadata...
But this is another story.